R/check-ages-over-90.R

```diff
@@ -55,8 +55,8 @@ check_ages_over_90 <- function(data, col = "ageDeath", strict = FALSE,
 }
 
 # Does the column (after removing non-numeric characters) contain any values
-# >90?
+# >90? NAs are not considered over 90 and will evaluate to FALSE.
 is_over_90 <- function(x) {
   col_numeric <- gsub("[^0-9]", "", x)
-  as.numeric(as.character(col_numeric)) > 90
+  (as.numeric(as.character(col_numeric)) > 90) %in% TRUE
```

Comment:
This broke my brain for a second. XD

Reply:
This is my solution for creating a vectorized version of `isTRUE()`

Reply:
It makes sense. Just took me a second to get it, haha.